src/comparisonData.tsx

```diff
@@ -7,20 +7,11 @@ export type ComparisonData = {
 
 export interface Comparison {
   id: number,
-  category: ComparisonCategory,
+  category: ComparisonCategory | undefined,
```

Comment:
We should take a look at this and see if this is truly how this needs to be handled.